cypress/integration/models/businessservices.ts

```diff
@@ -0,0 +1,149 @@
+import {
+    controls,
+    businessservices,
+    tdTag,
+    trTag,
+    button,
+    createNewButton,
+} from "../types/constants";
+import { navMenu, navTab } from "../views/menu.view";
+import {
+    businessServiceNameInput,
+    businessServiceDescriptionInput,
+    businessServiceOwnerSelect,
+} from "../views/businessservices.view";
+import {
+    confirmButton,
+    editButton,
+    deleteButton,
+    successAlertMessage,
+} from "../views/commoncontrols.view";
+import {
+    clickByText,
+    inputText,
+    click,
+    selectItemsPerPage,
+    submitForm,
+    cancelForm,
+    selectFormItems,
+    checkSuccessAlert,
+} from "../../utils/utils";
+import * as faker from "faker";
+
+export class BusinessServices {
+    businessServiceName;
+    businessServiceDescription;
+
+    protected static clickStakeholdergroups(): void {
+        clickByText(navMenu, controls);
+        clickByText(navTab, businessservices);
+    }
+
+    protected fillName(name: string): void {
+        inputText(businessServiceNameInput, name);
+    }
+
+    protected fillDescription(description: string): void {
+        inputText(businessServiceDescriptionInput, description);
+    }
+
+    protected selectOwner(owner: string): void {
+        selectFormItems(businessServiceOwnerSelect, owner);
+    }
+
+    getBusinessServiceName(): string {
```

Comment:
I made change for this approach, as you didn't want to set this.classvar name, you can update this to -

`
getBusinessServiceName(): string {
    return faker.company.companyName();
}
`